scripts/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -64,7 +63,23 @@ export default () => {
       if (!variation.specification) {
         const variationName = variation.name.replace('.ts', '');
         const definitionName = `${variationName}Definition`;
-        const definition = behaviorDefinitions[definitionName];
+
+        const absPathToBehaviorDefinition = path.normalize(
+          path.join(
+            require.resolve(`@fluentui/a11y-testing`),
+            '../',
+            'definitions',
+            behaviorName,
+            `${definitionName}.ts`,
+          ),
+        );
+        // delete require cache only works for absolute file path
+        delete require.cache[absPathToBehaviorDefinition];
+        let definition;
+        try {
+          // behavior definition file may not exist for components that haven't migrate to using a11y-testing
```

Comment:
if the file does not exist it throws the exception right ?

Reply:
right

Reply:
I would stick this comment in the catch block :)